spec/lib/capistrano/svn_spec.rb

```diff
@@ -59,6 +61,18 @@ module Capistrano
         context.expects(:execute).with(:svn, :update, '--username someuser', '--password somepassword')
         context.expects(:fetch).twice.with(:svn_username).returns('someuser')
         context.expects(:fetch).twice.with(:svn_password).returns('somepassword')
+        context.expects(:fetch).once.with(:svn_revision).returns(nil)
+
+        subject.update
+      end
+    end
+    
+    describe "#update_specific_revision" do
+      it "should run svn update and update to a specific revision" do
+        context.expects(:execute).with(:svn, :update, '--username someuser', '--password somepassword', '--revision 12345')
+        context.expects(:fetch).twice.with(:svn_username).returns('someuser')
+        context.expects(:fetch).twice.with(:svn_password).returns('somepassword')
+        context.expects(:fetch).once.with(:svn_revision).returns('12345')
```

Comment:
I think this should be `context.expects(:fetch).twice`.

… based on the [Travis build](https://travis-ci.org/capistrano/capistrano/builds/99255906) which is showing the failing test:

```
Failures:
  1) Capistrano::Svn::DefaultStrategy#update_specific_revision should run svn update and update to a specific revision
     Failure/Error: subject.update
     Mocha::ExpectationError:
       unexpected invocation: #<#<Class:0x00000003740b78>:0x3740b28>.fetch(:svn_revision)
       unsatisfied expectations:
       - expected exactly once, invoked twice: #<#<Class:0x00000003740b78>:0x3740b28>.fetch(:svn_revision)
       - expected exactly once, not yet invoked: #<#<Class:0x00000003740b78>:0x3740b28>.execute(:svn, :update, '--username someuser', '--password somepassword', '--revision 12345')
       satisfied expectations:
       - expected exactly twice, invoked twice: #<#<Class:0x00000003740b78>:0x3740b28>.fetch(:svn_password)
       - expected exactly twice, invoked twice: #<#<Class:0x00000003740b78>:0x3740b28>.fetch(:svn_username)
     # ./lib/capistrano/scm.rb:51:in `fetch'
     # ./lib/capistrano/svn.rb:12:in `svn'
     # ./lib/capistrano/svn.rb:32:in `update'
     # ./spec/lib/capistrano/svn_spec.rb:77:in `block (3 levels) in <module:Capistrano>'
```